### PR TITLE
feat(frontend): Audio player で波形を表示するように

### PR DESCRIPTION
**Pull request by mattyatea**
## Checklist
- [ ] Read the [contribution guide](https://github.com/misskey-dev/misskey/blob/develop/CONTRIBUTING.md)
- [ ] Test working in a local environment
- [ ] (If needed) Add story of storybook
- [ ] (If needed) Update CHANGELOG.md
- [ ] (If possible) Add tests

### CODE REVIEW

File: packages/frontend/src/components/MkAudioVisualizer.vue [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+</div>
+</template>
+
+<script lang="ts" setup>
+import { onMounted, onUnmounted, shallowRef, nextTick, ref } from 'vue';
+import * as Misskey from 'misskey-js';
+import * as THREE from 'three';
+import vertexShader from '../../node_modules/webgl-audiovisualizer/audiovisial-vertex.shader?raw';
+import fragmentShader from '../../node_modules/webgl-audiovisualizer/audiovisial-fragment.shader?raw';
+import circleMask from '../../node_modules/webgl-audiovisualizer/circlemask.png';
```
Comment (u1-liquid) [inline packages/frontend/src/components/MkAudioVisualizer.vue:12]:
```suggestion
import vertexShader from 'webgl-audiovisualizer/audiovisial-vertex.shader?raw';
import fragmentShader from 'webgl-audiovisualizer/audiovisial-fragment.shader?raw';
import circleMask from 'webgl-audiovisualizer/circlemask.png';
```
こうじゃ参照できないんだっけ？

Comment (mattyatea) [reply packages/frontend/src/components/MkAudioVisualizer.vue:12]: 無理そう

Comment (mattyatea) [reply packages/frontend/src/components/MkAudioVisualizer.vue:12]: image

Comment (u1-liquid) [reply packages/frontend/src/components/MkAudioVisualizer.vue:12]: じゃあ仕方ないか

**Pull request merged**